build.sh

```diff
@@ -178,8 +178,9 @@ if (( NUMARGS == 0 )) || hasArg rmm; then
     cd "${REPODIR}/python"
     export INSTALL_PREFIX
     echo "building rmm..."
+
     if [[ ${CUDA_MALLOC_ASYNC_SUPPORT} == OFF ]]; then
-        python setup.py build_ext_no_async --inplace
+        python setup.py build_ext --inplace -- -DCUDA_MALLOC_ASYNC_SUPPORT=OFF
```

Comment:
```suggestion
        python setup.py build_ext --inplace -DCUDA_MALLOC_ASYNC_SUPPORT=OFF
```

Reply:
Actually you need the extra `--`. In general, bash scripts treat `--` as a way to terminate a list of options. Invoking a setup.py script using scikit-build consists of three separate layers of execution: 1) the setup.py script itself, 2) the CMake command invocations, and 3) the underlying build tool (e.g. Ninja or make) invocation. You can specify options to each of these three steps using a suitable number of separators:
```
python setup.py install --SETUPTOOLS_OPTIONS -- --CMAKE_OPTION -- --NINJA_OPTION
```
In this case, we're passing a CMake option, so we need one `--`.